data/maps/map_keys.ini

```diff
@@ -17,6 +17,7 @@ tower of frozen shadow = frozenshadow
 icewell keep = thurgadinb
 the feerrott = feerrott
 ruins of sebilis = sebilis
+old sebilis = sebilis
```

Comment:
Do all of these change at some point in the P99 timeline and Green just isn't there yet? Or are they all broken on Blue as well?